src/nnet3/nnet-cudnn-simple-component.cc

```diff
@@ -294,6 +289,27 @@ void CuDNN3DConvolutionComponent::InitFromConfig(ConfigLine *cfl) {
   ok = ok && cfl->GetValue("filt-y-step", &filt_y_stride);
   ok = ok && cfl->GetValue("filt-z-step", &filt_z_stride);
 
+  // optional argument
```

Comment:
I change --input-num-filters into an optional field of the config line because i want to make the config line compatible with the original component. For the original component, the input-num-filters is always assumed to be one